### DISCUSSION TITLE
replace whiptail with gum?

### DISCUSSION THREAD

**Discussion by midzelis** [tteck/Proxmox#1834] [General]
Start started checking out these scripts, and so far I like them. However, using whiptail for input make the background super-blue and hurts my eyes! ;-) 

Have you seen gum? https://github.com/charmbracelet/gum

It seems like it would be very close to a drop in replacement.

**Comment by tteck** [comment]
I find gum to be interesting, although, in my view, it doesn't offer the same level of functionality as whiptail.
